.github/workflows/publish-operator-for-e2e-tests.yml

```diff
@@ -0,0 +1,60 @@
+name: publish-operator-for-e2e-tests
+on:
+  pull_request_target
+
+env:
+  GOPATH: /tmp/go
+  GO_VERSION: 1.15.x
```

Comment:
just curious: why 1.15 and not 1.16?

Reply:
yeah, this is still leftover from operator-sdk 1.7. I'll changed that

Reply:
done fa0d54c

Reply:
thanks!